source/_posts/2016-10-01-Insights-from-NSSpain.md

```diff
@@ -0,0 +1,58 @@
+
+**Burnout**   
+by [Marin Todorov](https://twitter.com/icanzilb) ([blogpost](https://medium.com/@marin.todorov/burnout-awareness-at-nsspain-6b852b1222d4))  
+
+The organisers also prepared a round of lightning talks and I was really impressed by Marin Todorov's. Marin is a successful iOS developer, instructor and book author, probably one of the most appreciated in our community. He gave a talk about burnout. He showed that burnout can happen to everyone, and we need to be better at preventing it and at helping others who are dealing with it. Burnout is more common than we want to admit in our industry, and seeing well-known developers draw attention to it and talk about it will hopefully make us more aware of burnout and help us prevent it for ourselves and for our friends and colleagues.
+
+
+## Dominik Hadl
+
+The power of Swift is clearly visible everywhere on the internet, but now it is even more obvious since 90% of talks at conferences contain Swift code examples. What's even more, serious talks about server-side Swift and people using it proves how fast is the language moving forward and gaining attention even from non-iOS developers. 
+
+What was interesting to me was that there were also quite a few talks showing various problems or issues without solutions - which seemed like a very good idea both for us to be able to avoid getting into a situation like that or the speakers who could get advice from other developers present in the audience.
+
+Workshop and talk by [Jorge Ortiz](https://twitter.com/jdortiz) was especially enlightening and made it very easy to grasp a difficult topic like **Clean Architecture** and proper dependency injection in Swift. We've gained some great ideas and with some modifications we are applying them to our current process and projects.
+
+**Karaoke talks** were a unique and extremely entertaining concept which I have never seen at any other conference. Very similar to lightning talks except for one major difference - the speaker did not know nor see his slides beforehand and had to come up with the topic as they were being displayed. This resulted in very funny and non-sensical talks with a bit of geeky developer humour.
+
+And last but not least, it was in Spain - land of tapas, vino and mañana! ☀️🇪🇸
+
+## Summary
+
+What I like about conferences is that when get back, you just want to do things, try all the stuff you learned about, play with new frameworks and SDKs, you feel full of energy, very motivated and very productive. 
+
+Another great thing about NSSpain is that the city it's in, Logroño, is the capital of the Rioja region, a world famous wine region. And the day after NSSpain starts the 1-week long San Mateo wine festival. 🍷 
```

Comment:
It kind of feels like it's missing a paragraph at the end, it seems a bit unfinished. How about switching the two paragraphs and adding a bit more to them? Something like:

> Another great thing about NSSpain is that the city it's in, Logroño, is the capital of the Rioja region, a world famous wine region. And the day after NSSpain starts the 1-week long San Mateo wine festival 🍷, where NSSpain participants can join the festivities and enjoy the local products and atmosphere.
> 
> Probably the best part about conferences is that when you get back, you just want to do things, try all the stuff you learned, play with new frameworks and SDKs, you feel full of energy, very motivated and very productive. We feel that our team has gained a lot by taking part in NSSpain, and we'll try to keep the practice of attending conferences for the following years.
> 

But with this change, it looks like the first paragraph about San Mateo shouldn't belong under the "Summary" headline. 
What do you think?